src/containers/donation/index.css

```diff
@@ -0,0 +1,171 @@
+body {
+  width: 100%;
+}
+
+.subTitle,
```

Comment:
so, in general, we rarely have rules like this with so many selectors. if htis is happening, you just move it to the top-levle. for example, `.pageMarings` would do the same thing.

Reply:
I deleted .pageMargins after I found it''s useless, so I can rename it to secondaryTitle, it might look more reasonable